scripts/dev_start.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 ip link add dev wg-firezone type wireguard
-ip address add dev wg-firezone 10.3.2.1/24
-ip -6 address add dev wg-firezone fd00::3:2:1/120
-ip link set up dev wg-firezone
+ip address replace dev wg-firezone 10.3.2.1/24
+ip -6 address replace dev wg-firezone fd00::3:2:1/120
+ip link set mtu 1280 up dev wg-firezone
```

Comment:
Standardizing on 1280 to standardize on lowest common denominator (this is the minimum) and prevent any possible MTU issues in different environments.